.travis.yml

```diff
@@ -1,13 +1,24 @@
 language: go
 dist: xenial
 go:
-  - 1.12.x
   - 1.13.x
+  - 1.14.x
```

Comment:
Maybe remove 1.14, as k8s skipped it because of https://github.com/kubernetes/kubernetes/issues/92521.